src/utility.py

```diff
@@ -20,4 +21,4 @@ def template(values):
     for item in items:
         window.addstr(y_pos, x_pos, template(item.values()))
         y_pos += 1
-    window.addstr(y_pos, x_pos, separator)
\ No newline at end of file
+    window.addstr(y_pos, x_pos, separator)
```

Comment:
Bitte nicht die bestehenden Funktionen in der utility.py das ist im default Branch des Hauptprojekts schon geschehen und ich kann sonst nicht mergen

Reply:
Das hat der autopep durchlauf gemacht

Reply:
ok das mache ich dann jetzt noch